src/Controls/src/Core/Compatibility/Handlers/Shell/Android/SearchHandlerAppearanceTracker.cs

```diff
@@ -270,6 +270,9 @@ protected virtual void Dispose(bool disposing)
 				if (_searchHandler != null)
 				{
 					_searchHandler.PropertyChanged -= SearchHandlerPropertyChanged;
+				}
+				if (_editText != null)
```

Comment:
LGTM! Did you find a related issue? Can you describe the steps to reproduce it? I'd like to include a relevant test with the changes.

Reply:
@jsuarezruiz No, I don't find a direct issue related to this.

I was just investigating to see if I can find the reason of the leaks and found this.

It's just a defensive check to ensure that the _editText is not null before attempting to remove it from the FocusChange event.